src/main/java/dev/slimevr/bridge/ProtobufBridge.java

```diff
@@ -212,14 +216,17 @@ public void addSharedTracker(ShareableTracker tracker) {
 		if (sharedTrackers.contains(tracker))
 			return;
 		sharedTrackers.add(tracker);
-		TrackerAdded.Builder builder = TrackerAdded.newBuilder().setTrackerId(tracker.getTrackerId()).setTrackerName(tracker.getDescriptiveName()).setTrackerSerial(tracker.getName()).setTrackerRole(tracker.getTrackerRole().id);
+		TrackerAdded.Builder builder = TrackerAdded.newBuilder().setTrackerId(tracker.getTrackerId())
+				.setTrackerName(tracker.getDescriptiveName()).setTrackerSerial(tracker.getName())
```

Comment:
See prior comment about newline wrapping behavior, this is another example but this time with chained function calls instead of function arguments. I would prefer that we do a newline per chained call when needing to wrap lines,  but I'd like to hear y'alls opinion.

Reply:
Same answer as the previous comment.